document/core/valid/types.rst

```diff
@@ -279,6 +279,8 @@ Limits
      \vdashlimitsmatch \{ \LMIN~n_1, \LMAX~m_1 \} \matcheslimits \{ \LMIN~n_2, \LMAX~m_2 \}
    }
 
+.. note::
+  In this context, read :math:`\matcheslimits` as “is the same type as or a subtype of”.
```

Comment:
Makes sense to point this out, but given that the subtyping symbol is used for all relations in this section, this seems like the wrong place to put this note. Instead of this note, I'd suggest changing line 243 to the following:
```
In some cases, this allows for a simple form of subtyping (written ":math:`\matches`" formally), as defined here.
```